cmd/network/verification.go

```diff
@@ -68,24 +80,30 @@ func NewCmdValidateEgress() *cobra.Command {
-	validateEgressCmd.Flags().StringVar(&e.subnetId, "subnet-id", "", "(optional) private subnet ID override, required if not specifying --cluster-id")
-	validateEgressCmd.Flags().StringVar(&e.securityGroupId, "security-group", "", "(optional) security group ID override for osd-network-verifier, required if not specifying --cluster-id")
-	validateEgressCmd.Flags().StringVar(&e.caCert, "cacert", "", "(optional) path to cacert file to be used with https requests being made by verifier")
-	validateEgressCmd.Flags().BoolVar(&e.noTls, "no-tls", false, "(optional) if provided, ignore all ssl certificate validations on client-side.")
-	validateEgressCmd.Flags().StringVar(&e.region, "region", "", "(optional) AWS region")
-	validateEgressCmd.Flags().BoolVar(&e.debug, "debug", false, "(optional) if provided, enable additional debug-level logging")
-
-	// If a cluster-id is specified, don't allow the foot-gun of overriding region
-	validateEgressCmd.MarkFlagsMutuallyExclusive("cluster-id", "region")
+	validateEgressCmd.Flags().StringVarP(&e.ClusterId, "cluster-id", "-c", "", "(optional) OCM internal/external cluster id to run osd-network-verifier against.")
+	validateEgressCmd.Flags().StringVar(&e.SubnetId, "subnet-id", "", "(optional) private subnet ID override, required if not specifying --cluster-id")
+	validateEgressCmd.Flags().StringVar(&e.SecurityGroupId, "security-group", "", "(optional) security group ID override for osd-network-verifier, required if not specifying --cluster-id")
+	validateEgressCmd.Flags().StringVar(&e.Cacert, "cacert", "", "(optional) path to cacert file to be used with https requests being made by verifier")
+	validateEgressCmd.Flags().BoolVar(&e.NoTls, "no-tls", false, "(optional) if provided, ignore all ssl certificate validations on client-side.")
+	validateEgressCmd.Flags().StringVar(&e.Region, "region", "", "(optional) AWS Region")
+	validateEgressCmd.Flags().StringVar(&e.HttpProxy, "http-proxy", "", "(optional) manually specify http proxy")
+	validateEgressCmd.Flags().StringVar(&e.HttpsProxy, "https-proxy", "", "(optional) manually specify https proxy")
+	validateEgressCmd.Flags().StringVar(&e.KmsKeyId, "kms-key-id", "", "(optional) manually specify an AWS KMS Key ID to encrypt the EC2 instance's volume with")
+	validateEgressCmd.Flags().StringVar(&e.AmiId, "ami-id", "", "(optional) manually specify the AMI to run")
+	validateEgressCmd.Flags().StringVar(&e.InstanceType, "instance-type", defaultInstanceType, "(optional) manually the AWS EC2 instance type")
+	validateEgressCmd.Flags().IntVar(&e.Timeout, "timeout", defaultTimeout, "(optional) manually specify timeout in seconds for individual egress health checks")
+	validateEgressCmd.Flags().BoolVarP(&e.Debug, "debug", "v", false, "(optional) if provided, enable additional debug-level logging")
+
+	// If a cluster-id is specified, don't allow the foot-gun of overriding Region
+	validateEgressCmd.MarkFlagsMutuallyExclusive("cluster-id", "Region")
```

Comment:
Yeah, that makes sense. For subnet-id/security-group I like keeping the option of overriding (in case you want to manually try a specific subnet/security-group), but I think making them exclusive for all the proxy configuration is a good idea for example. I'll add this!